### ISSUE TITLE
2.2

### ISSUE THREAD

**Issue by om3ga6400** [HJfod/AwesomeLoadAnimation#1]
is this dead or are you going to port it

**Comment by Cosmella-v** [comment]
@OM3GA6400 Hjfod would prob not port it, however i would like to see what this did look like in 2.1? (i might fork and port it if it looks like something i could do)

**Comment by om3ga6400** [comment]
> @OM3GA6400 Hjfod would prob not port it, however i would like to see what this did look like in 2.1? (i might fork and port it if it looks like something i could do)

yes please do that, i would do it myself but idk how to port stuff

**Comment by Cosmella-v** [comment]
@OM3GA6400 the problem is I don't know what the mod originally looked like and it has 2.1 memory addresses

**Comment by om3ga6400** [comment]
> @OM3GA6400 the problem is I don't know what the mod originally looked like and it has 2.1 memory addresses 

idk if this will work but you probably could try to compile it for 2.1 geode and run it on 2.1 to see what it dose
